eth_pydantic_types/hash.py

```diff
@@ -7,11 +7,19 @@
     with_info_before_validator_function,
 )
 
-from eth_pydantic_types.hexbytes import HexBytes
+from eth_pydantic_types.hex import HexBytes
 from eth_pydantic_types.serializers import hex_serializer
 from eth_pydantic_types.validators import validate_bytes_size
 
 
+def get_hash_pattern(size: int) -> str:
+    return f"^0x[a-fA-F0-9]{{{size}}}$"
```

Comment:
question: Should this be x2?  `Hash` help text suggests it's "byte-size" and this is hex string length.  So `Hash1` would use `get_hash_pattern(1)` would be a 4-bit hex string, not 1 byte.  Intentional?

Reply:
yes i had a bunch of mistakes with this, but it is all cleared up now and i added tests. it got a litle confusing for a bit, but ya, you are correct, thank youuuu